### ISSUE TITLE
Resources?

### ISSUE THREAD

**Issue by esoterra** [lann/wasmtime-component-serde#1]
Do resources become some sort of opaque integer/string id in a system like this?

**Comment by lann** [comment]
Yeah I would imagine so. I don't know that there is any one encoding that would make sense for every application or even every resource within an application. For this crate in particular I could imagine allowing for some kind of callbacks for (de)serializing handles.
